doc/lua_api.txt

```diff
@@ -617,6 +617,18 @@ Result is more like what you'd expect if you put a color on top of another
 color, meaning white surfaces get a lot of your new color while black parts
 don't change very much.
 
+#### `[item:<modname>:<itemname>:[W]x[H]
+
+Renders an inventory image for a given item to texture.
+Width & height are optional and specify the resulting resolution:
+
+* If `[item:<modname>:<itemname>` is used, width & height default to `item_rtt_speedup_resolution` (client setting)
+* If `[item:<modname>:<itemname>:W` is used, width & height are set to W
+* If `[item:<modname>:<itemname>:WxH` is used, width is set to W & height is set to H
```

Comment:
Item names may be in this format, but it's not enforced. `[item:air` is a valid item. Thus consider using `,` as second argument delimiter, despite being inconsistent to other modifiers.

Reply:
Yeah, I'm currently considering using `[item:<serialized_itemstack>:WxH`